src/main/java/org/eluder/coveralls/maven/plugin/util/Md5DigestInputStream.java

```diff
@@ -39,6 +39,6 @@ public Md5DigestInputStream(final InputStream stream) throws NoSuchAlgorithmExce
     }
 
     public String getDigestHex() {
-        return DatatypeConverter.printHexBinary(getMessageDigest().digest());
+        return new String(Hex.encodeHex(getMessageDigest().digest(),false));
```

Comment:
@sesuncedu couldn't we use https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Hex.html#encodeHexString-byte:A- ?

Reply:
The reason for the failing ci is a missing space after the comma, using the encodeHexString method would be nicer though IMO.

Reply:
@sesuncedu ping